docs/vectorizer-api-reference.md

```diff
@@ -256,41 +256,50 @@ The embedding functions are:
 
 ### ai.embedding_openai
 
-You use the `ai.embedding_openai` function to use an OpenAI model to generate embeddings.
+You call the `ai.embedding_openai` function to use an OpenAI model to generate embeddings.
 
 The purpose of `ai.embedding_openai` is to:
 - Define which OpenAI embedding model to use.
 - Specify the dimensionality of the embeddings.
 - Configure optional parameters like the user identifier for API calls.
-- Set the name of the environment variable that contains the OpenAI API key.
+- Set the name of the [environment variable that holds the value of your OpenAI API key][openai-use-env-var].  
 
 #### Example usage
 
-This function is used to create an embedding configuration object that is passed as an argument to [ai.create_vectorizer](#create-vectorizers):
+Use `ai.embedding_openai` to create an embedding configuration object that is passed as an argument to [ai.create_vectorizer](#create-vectorizers):
 
-```sql
-SELECT ai.create_vectorizer(
-    'my_table'::regclass,
-    embedding => ai.embedding_openai(
-      'text-embedding-3-small', 
-      768, 
-      chat_user => 'bob',
-      api_key_name => 'MY_OPENAI_API_KEY_NAME'
-    ),
-    -- other parameters...
-);
-```
+1. Set the value of your OpenAI API key [in an environmental variable][openai-set-key]. For example:
+
+   ```shell
+   echo "export OPENAI_API_KEY='yourkey'" >> ~/.zshrc
+   source ~/.zshrc
+   ```
+   
+2. In your vectorizer app, create your OpenAI configuration: 
```

Comment:
This is a database query. So you technically run it in your database. Additionally, you are not creating an OpenAI configuration here but a Vectorizer with OpenAI as embedding provider.

Reply:
Updated.